_locales/en/messages.json

```diff
@@ -96,10 +96,18 @@
 		"message": "Tab size",
 		"description": "Label for the text box controlling tab size option for the style editor."
 	},
+	"cm_theme": {
+		"message": "Theme",
+		"description": "Label for the style editor's CSS theme."
+	},
 	"dbError": {
 		"message": "An error has occurred using the Stylish database. Would you like to visit a web page with possible solutions?",
 		"description": "Prompt when a DB error is encountered"
 	},
+	"default": {
```

Comment:
It's likely the translation of "default" will be different for different things in other languages. Make this specific to the theme default.